confgenerator/testdata/invalid/windows/logging-receiver_windows_event_log_type_receiver_version_not1_unsupported_channel_characters/golden/error

```diff
@@ -0,0 +1,9 @@
+[21:7] "channels[0]" (a,b) contains an invalid character: ,
```

Comment:
I suppose client who wrongly input channels like this was going to do below stuff: 
`channels: [a,b]` 
or 
```
channels:
- a
- b
```
Can we modify the top error message from `"channels[0]" (a,b) contains an invalid character: ,` to  something obvious so that they know they need to put treat channels as a yaml list.

Reply:
The yaml library already enforces that it's a list and produces a different error if a non-list is supplied. This error is attempting to catch someone adding a literal `,` in the channel name, which isn't allowed.